src/sentry/api/endpoints/filechange.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from sentry.api.base import DocSection
+from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.models import CommitFileChange, Release, ReleaseCommit
+from rest_framework.response import Response
+
+
+class CommitFileChangeEndpoint(ProjectEndpoint):
```

Comment:
I think this should probably not inherit from the project endpoint since we'll want to move it to the organization path anyway. Instead, you could use the `OrganizationReleasesBaseEndpoint` and then check permissions similarly to https://github.com/getsentry/sentry/pull/5002 (which I'll merge once I figure out what is wrong w tests... tricky since the logs aren't available right now). But you would want to do something like:

```python
try:
    release = Release.objects.get(
        organization_id=organization.id,
        version=version,
    )
except Release.DoesNotExist:
    raise ResourceDoesNotExist

if not self.has_release_permission(request, organization, release):
    raise PermissionDenied
```
unfortunately that `has_release_permission` won't exist til i merge that though.